Docker.md

```diff
@@ -0,0 +1,49 @@
+# PegNet Docker
```

Comment:
Could you add a note on this file on how to edit the config file with this set up?

Reply:
editing the file should `defaultconfig.ini` should work, we can do it another way

Copy the `defaultconfig.ini` > `myconfig.ini`

and we can change the volume part to
```
volumes:
      - ./myconfig.ini:/root/.pegnet/defaultconfig.ini:ro
``` 

Maybe this is a better solution, what do you thing @PaulBernier ?

Reply:
Yep I think it's better. And add a line in the doc explaining that the config file is linked to the host file and that will be neat!

Reply:
will do, (will also add this to .gitignore)